pulp_container/tests/functional/api/test_tagging_images.py

```diff
@@ -144,7 +144,7 @@ def test_03_tag_second_image_with_same_tag(self):
         This test checks if a new repository version was created with a new content added
         and the old removed.
         """
-        manifest_a = self.get_manifest_by_tag("manifest_a")
+        manifest_a = self.get_manifest_by_tag("new_tag")
```

Comment:
I thought, they should result in the very same manifest if the tests above were successful. Did i miss something?

Reply:
Yes, they should be the same manifests. I am curious about the real difference between manifest_a and deleted_manifest - the second change in this PR